demo/custom.js

```diff
@@ -407,9 +407,14 @@ shakaDemo.Custom = class {
       const licenseServerURL = licenseServerUrlInput.value;
       const customDRMSystem = customDrmSystemInput.value;
       if (licenseServerURL) {
+        const filterFairplay =
```

Comment:
I don't like this solution.  Instead of filtering out fairplay from the loop when we add it in custom, I think it would be best if custom suppressed the entire loop.  Let's move the loop to the else clause of if (customDRMSystem).